docs/policies.md

```diff
@@ -140,8 +140,8 @@ Each member will be reimbursed **up to $3,000 in any rolling 3-year period** to
 
 **Provided to:** Members only
 
-As a remote organization, Catalysters rarely see one another in person. To encourage folks to bridge the virtual divide, each member may request reimbursement for up to **$200 per year** for travel and food expenses related to visiting another member of the coop. These expenses must be _marginal and additional_ to existing plans where the primary purpose of the expenses is visiting another co-oper . Receipts must be submitted through the expense reimbursement form for approval by Steve and reimbursement.
+As a remote organization, Catalysters rarely see one another in person. To encourage folks to bridge the virtual divide, each member may request reimbursement for up to **$200 per year** for travel and food expenses related to visiting another member of the coop. These expenses must be _marginal and additional_ to existing plans where the primary purpose of the expenses is visiting another co-oper . Receipts must be submitted through the expense reimbursement form for approval and reimbursement.
```

Comment:
We could add a link to the [reimbursement form](https://docs.google.com/forms/d/e/1FAIpQLSf4r4H-bqHsR_QVt_lALdBDBH_PRi96W8kPo0eCK1r-5SkZBg/viewform).

Reply:
Good idea! I'll do that